pythonrepo/__init__.py

```diff
@@ -31,7 +31,7 @@
 	print(str((ImportErr.args)))
 	ImportErr = None
 	del ImportErr
-	raise ImportError(str("PythonRepo Failed to Import"))
+	raise ImportError(str("pythonrepo Failed to Import"))
```

Comment:
`pythonrepo` is a more consistent spelling (than _P_`ython`_R_`epo`) of the module name.